src/it/java/com/google/checkstyle/test/chapter4formatting/rule4832nocstylearray/NoCStyleArrayDeclarationsTest.java

```diff
@@ -0,0 +1,40 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code and other text files for adherence to a set of rules.
+// Copyright (C) 2001-2024 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+package com.google.checkstyle.test.chapter4formatting.rule4832nocstylearray;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.checkstyle.test.base.AbstractGoogleModuleTestSupport;
+
+// -@cs[AbbreviationAsWordInName] Test should be named as its chapter name.
+public class NoCStyleArrayDeclarationsTest extends AbstractGoogleModuleTestSupport {
```

Comment:
will removed the suppression once https://github.com/checkstyle/checkstyle/pull/15116#discussion_r1652192468 is merged

Reply:
I thinks this `CStyle` should be ok to hardcode at `getExtractedRuleName(String ruleName)` and known edge case that we simply do search/replace to `Cstyle` before ` return extractedRuleName.toString();`

Reply:
I pushed an update

Reply:
I'll remove the suppression once the update has been merged

Reply:
removed the suppression: